### PR TITLE
video link

### PR DESCRIPTION
**Pull request by NikhilTilak**
- Added the youtube video link .

- github doesnt allow embedding videos in the readmes so adding a screenshot as a workaround.

### CODE REVIEW

**Comment by diptanil-roy** [comment]
Can you make the screenshot smaller and centered? It's yuuuuuge!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # bookend
+[![video_ss](figures/video_screenshot.png)](https://www.youtube.com/watch?v=P1Sq7T9PvP0)
```
Comment (emapple) [inline README.md]: Is it possible to decrease the size of the figure in line? It's kind of big. This suggestion may work, I'm not sure.

```suggestion
[![video_ss](figures/video_screenshot.png =250x)](https://www.youtube.com/watch?v=P1Sq7T9PvP0)
```

**Review by emapple — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (NikhilTilak) [reply README.md]: Thanks. I used html tags to scale it intead of github MD.

Comment (emapple) [reply README.md]: Looks good

**Review by emapple — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # bookend
 
+<p align="center">
+  <a href="https://www.youtube.com/watch?v=P1Sq7T9PvP0">
+      <img width="460" height="300" src="figures/video_screenshot.png">
+  </a>
+</p>
 bookend is a classifier for text data. We have built a machine learning model to identify the author of a work given only a selection of text. We are continually adding to the model and improving it, so make sure to check back for updates!
```
Comment (diptanil-roy) [inline README.md:11]: Add one more new line here, maybe! For some reason, there is no gap between the two paragraphs in your branch.

Comment (NikhilTilak) [reply README.md:11]: done

**Review by diptanil-roy — APPROVED** [review]

**Pull request merged**